packages/client/src/v2-events/components/forms/FormFieldGenerator.tsx

```diff
@@ -688,43 +689,67 @@ function makeFormikFieldIdsOpenCRVSCompatible<T>(data: Record<string, T>) {
   )
 }
 
-export const FormFieldGenerator: React.FC<ExposedProps> = (props) => {
-  const intl = useIntl()
+export const FormFieldGenerator: React.FC<ExposedProps> = React.memo(
+  (props) => {
```

Comment:
Could you elaborate on why it's needed to wrap this in memo? Is it possible to verify initial values do not change unnecessarily outside of the view logic?

Reply:
I came up with trying this when using useEffect created an infinite loop. But that problem is solved. It doesn't have any effect now and removing it.